Dockerfile.ubi

```diff
@@ -69,10 +71,27 @@ RUN --mount=type=cache,target=/root/.cache/uv \
     uv pip install \
         -r requirements/cuda.txt
 
+## Mumba build #################################################################
+FROM python-cuda-base AS mamba-build
+ARG MAX_JOBS
+
+ENV MAX_JOBS=${MAX_JOBS}
+ENV MAMBA_FORCE_BUILD="TRUE"
+ENV MAMBA_FORCE_CXX11_ABI="FALSE"
+ENV HIP_ARCHITECTURES="gfx942"
+
+# Pre-install mamba-ssm into the cache so uv reuses it later
+RUN --mount=type=cache,target=/root/.cache/uv \
+    uv pip install torch packaging psutil setuptools wheel && \
```

Comment:
You're missing `--constraints requirements/cuda.txt` here. You might end up installing a different torch version than what we're using in vllm